sdk/nodejs/invoke.ts

```diff
@@ -34,10 +34,4 @@ export interface InvokeOptions {
      * invoke.
      */
     version?: string;
-
-    /**
-     * Invoke this function asynchronously.  If 'true' is passed in here, then an invoked function
-     * will only supply the `Promise<>` side of its result.
-     */
-    async?: boolean;
```

Comment:
note: we could also just deprecate this but keep it around to make the break less impactful for code that is already passing `{ async: true }`.  Note that the break here is minor given that it's just a TS error, not a runtime error if we remove this option.